src/bios.rs

```diff
@@ -12,6 +14,30 @@ use crate::packagesystem;
 // grub2-install file path
 pub(crate) const GRUB_BIN: &str = "usr/sbin/grub2-install";
 
+#[cfg(target_arch = "powerpc64")]
+fn target_device(device: &str) -> Result<Cow<str>> {
```

Comment:
Just one minor suggestion, not blocker, better to rename `target_device` to `get_prepboot_device()`, and move it to `src/blockdev.rs`? I am also OK to keep it here as it is only used on ppc64.